tests/UnitTests/ApplicationCore/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Xunit;
+
+namespace Microsoft.eShopWeb.UnitTests.ApplicationCore.Extensions
+{
+    public class StringExtensions
+    {
+        [Fact]
+        public void CorrectlySerializesAndDeserializesObject()
+        {
+            var foo = new Foo
+            {
+                Id = 7,
+                Name = "Test name",
+                Children = new[]
+                {
+                    new Bar(),
+                    new Bar(),
+                    new Bar()
+                }
+            };
+
+            var json = foo.ToJson();
+            var result = json.FromJson<Foo>();
+            Assert.Equal(foo, result);
+        }
+
+        [
+            Theory,
+            InlineData("{ \"id\": 9, \"name\": \"Another test\" }", 9, "Another test"),
+            InlineData("{ \"id\": 3124, \"name\": \"Ugh, this doesn't matter...\" }", 3124, "Ugh, this doesn't matter..."),
```

Comment:
Can you replace `"Ugh, this doesn't matter"` to something like `Another Tests 2`, or `Test Value 1`, something like that?